copy-track-files.ps1

```diff
@@ -21,17 +21,23 @@ param (
 # Import shared functionality
 . ./shared.ps1
 
-$defaultEditorConfigSettings = Get-Content -Path ".editorconfig"
 
 function Copy-Track-Files-For-Exercise ($ExerciseDirectory) {
     $exerciseName = (Get-Culture).TextInfo.ToTitleCase($ExerciseDirectory.Name).Replace("-", "")
+
+    $defaultEditorConfigSettings = Get-Content -Path ".editorconfig"
     $editorConfigSettings = $defaultEditorConfigSettings.Replace( "[*.cs]", "[${exerciseName}.cs]")
     $exerciseEditorConfigPath = Join-Path $ExerciseDirectory.FullName ".editorconfig"
 
     Set-Content -Path $exerciseEditorConfigPath $editorConfigSettings
 }
 
-function Copy-Track-Files {
+function Copy-TrackFile {
```

Comment:
```suggestion
function Copy-TrackFiles {
```

Reply:
Copy-TrackFilesForExercise works for the analyzer.